analyzer/plugin-api/src/main/java/jp/co/ntt/oss/heapstats/utils/HeapStatsUtils.java

```diff
@@ -227,19 +226,13 @@ public static void load() throws IOException, HeapStatsConfigException {
     }
 
     /**
-     * Get plugin list.
+     * Get plugin list by config.
      *
      * @return Plugin list.
      */
     public static List<String> getPlugins() {
-        List<String> pluginList = new ArrayList<>();
-        pluginList.add(LogController.class.getPackage().getName());
-        pluginList.add(SnapShotController.class.getPackage().getName());
-        pluginList.add(ThreadRecorderController.class.getPackage().getName());
-        pluginList.add(JVMLiveController.class.getPackage().getName());
-        pluginList.addAll(Arrays.asList(prop.getProperty("plugins", "").split(";")));
-
-        return pluginList.stream()
+        return Arrays.asList(prop.getProperty("plugins", "").split(";"))
+                .stream()
                 .map(s -> s.trim())
                 .filter(s -> s.length() > 0)
                 .distinct()
```

Comment:
This statement returns unique `pluginList`. However builtin plugins will be added at `MainWindowController#loadPlugin()`.
`distinct()` should be called at `loadPlugin()`.